presto-main/src/test/java/io/prestosql/server/protocol/TestQueryResultRows.java

```diff
@@ -65,7 +66,7 @@ public void shouldReturnSingleValue()
 
         assertThat((Iterable<? extends List<Object>>) rows).as("rows").isNotEmpty();
         assertThat(getAllValues(rows)).hasSize(1).containsOnly(ImmutableList.of(true));
-        assertThat(rows.getColumns()).containsOnly(column);
+        assertThat(rows.getColumns().get()).containsOnly(column);
```

Comment:
Use `orElseThrow()` for all of these `get()` calls as that's the new preferred method when the value presence is not previously checked (so the expected behavior of throwing is clear).